pandas/core/reshape/concat.py

```diff
@@ -25,9 +25,28 @@
 # ---------------------------------------------------------------------
 # Concatenate DataFrame objects
 
+FrameOrSeriesUnion = Union["DataFrame", "Series"]
```

Comment:
maybe we should put this in _typing as FrameOrSeries and rename FrameOrSeries as FrameOrSeriesT. @WillAyd